src/ert/resources/workflows/jobs/internal-gui/scripts/gen_data_rft_export.py

```diff
@@ -119,8 +115,7 @@ def run(
         output_file = workflow_args[0]
         trajectory_path = workflow_args[1]
         ensemble_list = None if len(workflow_args) < 3 else workflow_args[2]
-        _ = True if len(workflow_args) < 4 else workflow_args[3]
-        drop_const_cols = False if len(workflow_args) < 5 else workflow_args[4]
+        drop_const_cols = False if len(workflow_args) < 4 else workflow_args[3]
```

Comment:
This ends up being a breaking change for the workflow, which is used quite a bit. We should be a bit careful here.